packages/cli/src/collect/collect.js

```diff
@@ -80,6 +80,12 @@ function buildCommand(yargs) {
       default: 3,
       type: 'number',
     },
+    numberOfFiles: {
+      alias: 'f',
```

Comment:
`-f` feels a little too prominent for a fallback option we don't expect many folks to use, mind removing the alias?

Reply:
Yeah you are right about this one.